worker/src/services/IngestionService/index.ts

```diff
@@ -759,7 +852,7 @@ export class IngestionService {
         prompt_version: prompt?.version,
         created_at: Date.now(),
         updated_at: Date.now(),
-        event_ts: new Date(obs.timestamp).getTime(),
+        event_ts: new Date(obs.body?.startTime ?? obs.timestamp).getTime(),
```

Comment:
good one!

Reply:
above, we are setting the start_time like: 

```
 start_time: this.getMillisecondTimestamp(
          obs.body.startTime ?? obs.timestamp,
        ),

```
I think we should remove this and default to undefined. Can you also check whether this is for scores + traces the same?

Reply:
Good catch 😁 Fixed in https://github.com/langfuse/langfuse/pull/3978/commits/7ce2bb895e0b38d4276ab4bf7535869dac3b1369